ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -202,6 +203,10 @@ object Config {
               .get("indexer-input-mapping-parallelism")
               .map(_.toInt)
               .getOrElse(ParticipantIndexerConfig.DefaultInputMappingParallelism)
+            val indexerMaxInputBufferSize = kv
```

Comment:
Could you also add a corresponding CLI parameter to [the indexer benchmark config](https://github.com/digital-asset/daml/blob/main/ledger/indexer-benchmark/src/main/scala/ledger/indexerbenchmark/Config.scala#L54)?